public/test.html

```diff
@@ -134,12 +134,16 @@ <h4>Search for something</h4>
 
   <h3><a href="https://en.wikipedia.org/wiki/Geo_URI_scheme">Geo url scheme</a></h3>
   <ol>
-    <li><a href="geo:37.786971,-122.399677">geo:37.786971,-122.399677</a></li>
```

Comment:
Why did you delete these lines? I'm using them for testing.

Reply:
I just added a comment. This link is just below.